### PR TITLE
Upgrade phpunit

### PR DESCRIPTION
**Pull request by mvorisek**

### CODE REVIEW

**Comment by mvorisek** [comment]
Help wanted with the failing tests.

File: tests/WithDb/SelectTest.php [hunk 1]
```diff
@@ -17,7 +17,7 @@ protected function setUp(): void
         $this->c = Connection::connect($GLOBALS['DB_DSN'], $GLOBALS['DB_USER'], $GLOBALS['DB_PASSWD']);
 
         $pdo = $this->c->connection();
-        $pdo->query('CREATE TEMPORARY TABLE employee (id int not null, name text, surname text, retired bool, PRIMARY KEY (id))');
+        $pdo->query('CREATE TABLE employee (id int not null, name text, surname text, retired bool, PRIMARY KEY (id))');
```
Comment (mvorisek) [inline tests/WithDb/SelectTest.php]: @DarkSide666 hope only for tests ;-)

Comment (DarkSide666) [reply tests/WithDb/SelectTest.php]: yes just testing because I do not have pgsql on my local ready for tests

**Review by DarkSide666 — APPROVED** [review]

File: src/Connection_PgSQL.php [hunk 2]
```diff
@@ -22,6 +22,11 @@ class Connection_PgSQL extends Connection
     public function lastInsertID($m = null)
     {
         // PostGRE SQL PDO requires sequence name in lastInertID method as parameter
-        return $this->connection()->lastInsertID($m->sequence ?: $m->table . '_' . $m->id_field . '_seq');
+        try {
+            return $this->connection()->lastInsertID($m->sequence ?: $m->table . '_' . $m->id_field . '_seq');
+        } catch (\PDOException $e) {
+            // if no sequence defined (we do not always need it), then silence PDO exception
```
Comment (mvorisek) [inline src/Connection_PgSQL.php:28]: @DarkSide666 This is dangerous as exception may come from broken connection etc., you should definitelly check specific error code here. See https://www.php.net/manual/en/pdo.errorinfo.php - `->errorinfo[2]`.

Comment (DarkSide666) [reply src/Connection_PgSQL.php:28]: You're probably right here. I will make PR.

Comment (DarkSide666) [reply src/Connection_PgSQL.php:28]: https://github.com/atk4/dsql/pull/195

**Pull request merged**